community/php5/APKBUILD

```diff
@@ -339,7 +339,7 @@ doc() {
 
 common() {
 	pkgdesc="PHP Common Files"
-	provides="$pkgname-zlib"  # for backward compatibility
+	provides="$pkgname-zlib php-zlib"  # for backward compatibility
```

Comment:
probably "php-common" should be added here as well, so kinda `provides="$pkgname-zlib php-zlib php-common"`